### ISSUE TITLE
Not possible to add [APM Request Latency] via DistributionWidgetDefinition class widget

### ISSUE THREAD

**Issue by Bartket** [DataDog/datadog-api-client-python#580]
**Describe the bug**
No possibility to add apm_stats_query to the DistributionWidgetDefinition widget class.
apm_query=LogQueryDefinition does not make it possible to reproduce APM Request Latency distribution chart for the service.

Label the issue properly.
`severity/medium`

**To Reproduce**
1. Add DistributionWidgetDefinition widget to the Dashboard class as follows:
```python
            Widget(
                definition=DistributionWidgetDefinition(
                    title="Latency distribution",
                    title_size="16",
                    show_legend=False,
                    title_align=WidgetTextAlign("left"),
                    type=DistributionWidgetDefinitionType("distribution"),
                    xaxis=DistributionWidgetXAxis(max="p95"),
                    yaxis=DistributionWidgetYAxis(scale="linear"),
                    requests=[
                        DistributionWidgetRequest(
                            apm_stats_query=ApmStatsQueryDefinition(
                                env="env",
                                name="service_request",
                                primary_tag="",
                                row_type=ApmStatsQueryRowType("service"),
                                service="service",
                            ),
                        ),
                    ],
                ),
            ),
```

**Expected behavior**
Possibility to add apm_stats_query parameter and ApmStatsQueryDefinition class to the DistributionWidgetDefinition class is added.

**Screenshots**
![obraz](https://user-images.githubusercontent.com/22456734/132862324-5bdcb7b3-13e9-4fa8-a39b-f0d7a59bb443.png)
I Would like to have this option via Python API for the distribution widget which is not possible now.

**Environment and Versions (please complete the following information):**
OS: Ubuntu 20.04 LTS
Python: 3.8.10
datadog-api-client: 1.3.0

**Additional context**

**Comment by therve** [comment]
Hi, I believe it has been fixed with #628 . Thanks.

**Comment by Bartket** [comment]
@therve Cool, I managed to test and push to my repo, works as desired. Thanks!
